packages/shared/styles/components/SfCharacteristic.scss

```
@@ -0,0 +1,25 @@
+@import '../variables';
+
+$characteristic-margin-right: 20px;
+
+.sf-characteristic {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__icon {
+    display: flex;
+    align-items: center;
+    margin-right: $characteristic-margin-right;
+  }
+
+  &__title {
+    font-size: $font-size-small-desktop;
```

imo it should be a component variable:

```scss
$characteristic__title-font-size: $font-size-small-desktop !default;
```

```suggestion
    font-size: $characteristic__title-font-size;
```

---

wdyt?

---

Sure, I agree with you :)